dev-libs/libratbag/libratbag-0.13.ebuild

```diff
@@ -30,14 +32,13 @@ RDEPEND="
 	${PYTHON_DEPS}
 	dev-libs/libevdev
 	virtual/libudev
-"
-DEPEND="
-	${RDEPEND}
-	dev-lang/swig
+	$(python_gen_cond_dep 'dev-libs/gobject-introspection[${PYTHON_SINGLE_USEDEP}]')
```

Comment:
If depending on `dev-libs/gobject-introspection`, in this case, Python modules provided by `dev-libs/gobject-introspection` are not used, so drop `$(python_gen_cond_dep '` and `[${PYTHON_SINGLE_USEDEP}]')`.